content/docs/forms.md

```diff
@@ -64,29 +64,29 @@ class NameForm extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/VmmPgp?editors=0010)
+[**Prova-ho a CodePen**](https://codepen.io/gaearon/pen/VmmPgp?editors=0010)
 
-Since the `value` attribute is set on our form element, the displayed value will always be `this.state.value`, making the React state the source of truth. Since `handleChange` runs on every keystroke to update the React state, the displayed value will update as the user types.
+Ja que l'atribut `value` és afegit al nostre element del formulari, el valor mostrat sempre serà el de `this.state.value`, fent que l'estat de React sigui la font de la veritat. Ja que `handleChange` corre cada vegada que una tecla és oprimida per actualitzar l'estat de React, el valor mostrat serà actualitzat mentre l'usuari escriu.
```

Comment:
```suggestion
Ja que l'atribut `value` és assignat al nostre element del formulari, el valor mostrat sempre serà el de `this.state.value`, fent que l'estat de React sigui la font de la veritat. Ja que `handleChange` corre cada vegada que una tecla és oprimida per actualitzar l'estat de React, el valor mostrat serà actualitzat mentre l'usuari escriu.
```

Reply:
I think it should be
```suggestion
Ja que l'atribut `value` és assignat al nostre element del formulari, el valor mostrat sempre serà `this.state.value`, fent que l'estat de React sigui la font de veritat. Ja que `handleChange` corre cada vegada que una tecla és oprimida per actualitzar l'estat de React, el valor mostrat serà actualitzat mentre l'usuari escriu.
```
since the source is always the real one (aka "veritable") but not always the source of truth (aka "de veritat")